src/hooks/__tests__/useCalculateSwapPairs.ts

```diff
@@ -5,7 +5,15 @@ import { __test__ } from "../useCalculateSwapPairs"
 const { getTradingPairsForToken } = __test__
 
 const createTestToken = (name: string, isSynth?: boolean) => {
-  return new Token({ 1: "", 31337: "" }, 0, name, "", name, "", !!isSynth)
+  return new Token(
+    { 1: "", 3: "", 31337: "" },
```

Comment:
this can be generated programmatically from `Object.keys(ChainId)` so it won't need to be updated each time we add a new supported chain

Reply:
this comment was marked as resolved but wasn't addressed